### PR TITLE
Remove duplicate unit tests

### PR DESCRIPTION
**Pull request by hishamco**
/cc @sarahelsaig

### CODE REVIEW

File: test/OrchardCore.Tests/Data/ContentItemTests.cs [hunk 1]
```diff
@@ -45,19 +45,6 @@ public void ShouldSerializeParts()
             Assert.Equal("test", (string)contentItem2.Content.MyPart.Text);
         }
 
-        [Fact]
```
Comment (hishamco) [inline test/OrchardCore.Tests/Data/ContentItemTests.cs:48]: This equivalent to

https://github.com/OrchardCMS/OrchardCore/blob/ab07490a2d5ae54180d0af3db27c89287991e567/test/OrchardCore.Tests/Data/ContentItemTests.cs#L61-L72

File: test/OrchardCore.Tests/Email/EmailTests.cs [hunk 2]
```diff
@@ -73,20 +73,6 @@ public async Task SendEmail_WithDisplayName()
             await SendEmailAsync(message, "Your Name <youraddress@host.com>");
         }
 
-        [Fact]
```
Comment (hishamco) [inline test/OrchardCore.Tests/Email/EmailTests.cs:76]: This equivalent to

https://github.com/OrchardCMS/OrchardCore/blob/ab07490a2d5ae54180d0af3db27c89287991e567/test/OrchardCore.Tests/Email/EmailTests.cs#L9-L25

Comment (Piedone) [reply test/OrchardCore.Tests/Email/EmailTests.cs:76]: This has a more appropriate name though, since it's not about the To header, so please rename it.

**Review by Piedone — CHANGES_REQUESTED** [review]

File: test/OrchardCore.Tests/Email/EmailTests.cs (hunk 2 above)
Comment (hishamco) [reply test/OrchardCore.Tests/Email/EmailTests.cs:76]: I will use the other name

**Review by Piedone — APPROVED** [review]

**Pull request merged**